src/components/footer.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+interface FooterProps {
+  todos: Todo[];
+  filter: 'all' | 'active' | 'completed';
```

Comment:
```suggestion
  filter: Filter;
```
You should use enum everywhere for it